proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -30,6 +30,10 @@ service Msg {
   // Undelegate defines a method for performing an undelegation from a
   // delegate and a validator.
   rpc Undelegate(MsgUndelegate) returns (MsgUndelegateResponse);
+
+  // CancelUnbondingDelegation defines a method for performing canceling the unbonding delegation
+  // and delegate back to previous validator
+  rpc CancelUnbondingDelegation(MsgCancelUnbondingDelegation) returns (MsgCancelUnbondingDelegationResponse);
```

Comment:
Adding a message in consensus breaking. Let's add:
- a changelog entry in state-machine breaking
- bump the module consensus version in x/staking/module.go

Reply:
Okay